src/resources/parser/glb-parser.js

```diff
@@ -1308,6 +1308,24 @@ var createNodes = function (gltf, options) {
     return nodes;
 };
 
+var createScenes = function (gltf, nodes) {
```

Comment:
this only creates a root node for each scene, and just links up already created nodes.

Reply:
What's the purpose of the extra scene nodes?

Reply:
it's a grouping per scene. If glb contains multiple scenes (sample I have contains 4), that allows user to grab scenes out, otherwise how would they tell root nodes apart?

Reply:
Oh I see, scene.nodes is an array (duh).

In that case you probably only want to create a scene parent if it comprises more than one node (similar to what's done for the top-level node, which is only created when there is more than one scene). The reason is you don't want to create unnecessary nodes at the root of each model which the user doesn't expect and must then traverse.

Reply:
ok, if there is a single scene with a single node in it, I don't create a scene root anymore - so that's the case for normal characters (tested). But if there is more than 1 scene, I create wrappers allowing user to consistently find a scene by name if needed.

Reply:
sweet